src/DblParser/Lexer.mll

```diff
@@ -80,13 +80,23 @@ let tokenize_ident str =
   | None     -> YaccParser.LID str
 
 let num_regex = Str.regexp
-  {|^\(0[bB][01]*\|0[oO][0-7]*\|[0-9]*\|0[xX][0-9a-fA-F]*\)$|}
+  {|^\(0[bB][01]*\|0[oO][0-7]*\|[0-9]*\|0[xX][0-9a-fA-F]*\)[lL]?$|}
 
 let tokenize_number pos str =
   if not (Str.string_match num_regex str 0) then
     Error.fatal (Error.invalid_number
       (Position.of_lexing (String.length str) pos)
       str)
+  else if String.length str > 0 &&
+    (str.[String.length str - 1] = 'l' ||
```

Comment:
In OCaml, the `l` suffix stands for `int32` literals. Maybe we should not use `l` here for similar reason?